analyzers/tests/SonarAnalyzer.UnitTest/CFG/Roslyn/ControlFlowBranchTest.cs

```diff
@@ -37,10 +41,17 @@ public class Sample
     public void Method(bool condition) { } // Empty, just Entry and Exit block
 }";
             var cfg = TestHelper.CompileCfg(code);
-            var entry = cfg.Blocks[0];
-            var exit = cfg.Blocks[1];
+            var entry = cfg.EntryBlock;
+            var exit = cfg.ExitBlock;
             entry.Kind.Should().Be(BasicBlockKind.Entry);
+            entry.Operations.Should().BeEmpty();
+            entry.BranchValue.Should().BeNull();
+            entry.OperationsAndBranchValue.Should().BeEmpty();
```

Comment:
This new lines shouldn't be here. It should be in `BasicBlockTest` file.

These assertions are here just to make sure we've picked up correct `Entry` and `Exit` blocks. But with the new properties `EntryBlock` and `ExitBlock` even these assertions can be deleted.